src/Navigation/NavigationFooter.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Compass } from '@styled-icons/fa-solid/Compass';
+import { MainInterface, ResponsiveInterface } from '../Utils/BaseStyles';
+import { ImplicitPropsInterface } from '../Utils/Hooks';
+import { Mixins } from '../Utils';
+import { Paragraph as P } from '../Text';
+
+export interface NavigationFooterProps
+    extends MainInterface,
+        ResponsiveInterface,
+        ImplicitPropsInterface,
+        React.HTMLAttributes<HTMLDivElement> {
+    url: string;
+    text: string;
+}
+
+export const NavigationFooter: React.FC<NavigationFooterProps> = ({
+    url,
+    text,
+    ...props
+}): React.ReactElement => (
+    <NavigationFooter_ {...props}>
```

Comment:
No more with this naming 'NavigationFooter_', what is this component doing? Be more descriptive, don't use '_'